src/components/popup/home/Tabs.tsx

```diff
@@ -70,15 +70,17 @@ const TabsWrapper = styled.div`
   filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.25));
 `;
 
-const StyledTab = styled.div<{ active?: boolean; tabId: number }>`
+const StyledTab = styled.button<{ active?: boolean; tabId: number }>`
   position: relative;
   height: 25px;
   font-weight: 500;
   font-size: 18px;
   line-height: 25px;
-  color: ${(props) =>
-    props.active ? props.theme.primaryText : props.theme.secondaryTextv2};
+  color: ${(props) => (props.active ? "white" : props.theme.secondaryTextv2)};
```

Comment:
```suggestion
  color: ${(props) => (props.active ? props.theme.primaryText : props.theme.secondaryTextv2)};
```

Because in Light Mode this ends up being white on white